spec/rspec-dns/have_dns_spec.rb

```diff
@@ -10,6 +10,26 @@ def stub_records(strings)
   allow(resolver).to receive_message_chain(:query, :answer).and_return(records)
 end
 
+def stub_records_authority(strings)
```

Comment:
Seems like two new methods duplicate quite a lot of code from `stub_records`.
How about adding a second parameter to stub_records and default it to `:answer`?